index.js

```diff
@@ -25,4 +40,63 @@ client.on('guildMemberAdd', member => {
 	channel.send(welcomes[Math.floor(Math.random() * welcomes.length)]);
 });
 
+client.on('message', message => {
+	if (message.channel.type !== 'dm' || message.author.bot) return;
```

Comment:
I've seen a common way that I've seen bots communicate is on the guild, and instead
of giving the channel name they just give #channel-name. This is slightly nicer
because the channel names will autocomplete and it will be a bit more usable.

Don't actually implement this, you've done a great job, just commenting

Reply:
Yes, well normally I'd do that but I wanted all the poll setup to be in a dm, and I didn't want a command like `!poll` sitting in our announcements channel until the setup is complete, so I wanted the flow to be initiated in a dm.